pages/mint-nft/index.vue

```diff
@@ -585,8 +610,16 @@ async function onISCNIDInput () {
       iscnOwner.value = owner
       step.value = 2
     } else if (iscnIdInput.value.startsWith('likenft')) {
-      const data = await $fetch(`${LCD_URL}/cosmos/nft/v1beta1/classes/${encodeURIComponent(iscnIdInput.value)}`)
-      if (!data) { throw new Error('INVALID_NFT_CLASS_ID') }
+      isRestockingNFT.value = true
+      const [data, classInfo] = await Promise.all([
+        $fetch(`${LCD_URL}/cosmos/nft/v1beta1/classes/${encodeURIComponent(iscnIdInput.value)}`),
+        $fetch(`${LIKE_CO_API}/likernft/book/store/${iscnIdInput.value}`)
+      ])
+      if (!data || !classInfo) {
+        isRestockingNFT.value = false
+        throw new Error('INVALID_NFT_CLASS_ID')
+      }
```

Comment:
`classInfo` can be `null` and throws 404 if the book is completely new and haven't be listed yet, which should be an expected case instead of error?